File: module-core/src/main/java/dto/MovieDTO.java
package dto;

import java.time.LocalDate;

import lombok.Data;

@Data
public class MovieDTO {
	private Long id;
	private String title;
	private LocalDate openDay;
	private Integer runningTime;
	private String thumbnail;
	private GenreDTO genre;
	private RatingDTO rating;
	private String plot;
}

File: src/main/kotlin/com/example/demo/application/dto/MovieDto.kt
package com.example.demo.application.dto

import java.time.LocalDateTime

data class MovieDto(
    val title: String,
    val rating: String,
    val releaseDate: LocalDateTime,
    val thumbnailUrl: String,
    val runningTime: Int,
    val genre: String,
    val screenings: List<ScreeningDto>
)

data class ScreeningDto(
    val cinemaName: String,
    val startTime: LocalDateTime
) 